assets/nf-core_style.css

```diff
@@ -0,0 +1,70 @@
+body {
```

Comment:
Not sure this belongs in here - looks like this is a remnant from an nf-core/rnaseq push?

Reply:
This is the css style for the Rmarkdown report 😄